src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs

```diff
@@ -25,10 +26,14 @@ internal class IntellisenseData : IIntellisenseData
         // $$$ This should probably be symbol and not just config. 
         protected readonly PowerFxConfig _powerFxConfig;
 
+        internal PowerFxConfig PowerFxConfig => _powerFxConfig;
+
         internal Features Features => _powerFxConfig.Features;
 
         private readonly IEnumStore _enumStore;
 
+        internal IEnumStore EnumStore => _enumStore;
```

Comment:
>IEnumStore EnumStore [](http://example.com/codeflow?start=17&length=20)

Why do we need two properties exposing same information? Why not make only one readonly property?